cogs/moveusers.py

```diff
@@ -1,7 +1,7 @@
 import discord
 from discord.ext import commands
 
-#Command that moves one user in a voice channel into another voice channel specified by the user
+# Command that moves one user in a voice channel into another voice channel specified by the user
```

Comment:
Typically let's not put in nonrelated changes but for this PR it's fine especially cuz this is just a formatting thing